### DISCUSSION TITLE
Theme icons

### DISCUSSION THREAD

**Discussion by francescoinblack** [bbidulock/icewm#732] [General]
Is there a way to understand by what order icewm takes icons from themes ?
For example I have:
#  Icon search path (colon separated)
IconPath="/usr/local/share/icons:/usr/local/share/pixmaps:/usr/share/icons:/usr/share/pixmaps:/usr/share/icewm/icons:/home/francesco/.icewm/iceicons-default-0.10.0/icons:/home/francesco/.icewm/icons:"
Bye
Francesco bat

**Comment by gijsbers** [comment]
Does this give more information?
https://ice-wm.org/man/icewm-preferences#paths

**Reply by francescoinblack** [reply]
I understand.
Thank you very much ;-)
